src/pages/gen2/reference/project-structure/index.mdx

```diff
@@ -101,7 +101,7 @@ defineBackend({
 });
 ```
 
-Backends can be extended using the [AWS Cloud Development Kit (CDK)](https://docs.aws.amazon.com/cdk/v2/guide/home.html). By leveraging CDK you can build using any AWS service, such as an Amazon S3 bucket that authenticated users have read and write access to. You can get started by installing AWS CDK:
+You can extend backends by using the [AWS Cloud Development Kit (AWS CDK)](https://docs.aws.amazon.com/cdk/v2/guide/home.html). With the CDK, you can build using any AWS service, such as an Amazon S3 bucket that authenticated users have read and write access to. To get started with the CDK, first install it:
```

Comment:
should we use "AWS CDK" when referring to CDK?

Reply:
According to the AWS service names wiki, we should spell out "AWS Cloud Development Kit (AWS CDK)" on first use. It's OK to use the extra-short name ("CDK") in service documentation when the extra-long name ("AWS Cloud Development Kit") or short name ("AWS CDK") has already been used in the same topic.

Reply:
that makes sense to me! resolving